modmesh/view.py

```diff
@@ -60,6 +62,30 @@ def _load():
     if enable:
         for name in _from_impl:
             globals()[name] = getattr(_vimpl, name)
+        # Try to find the PUI in thirdparty, if failed to find PUI
+        # modmesh will raise ImportError and terminate itself.
+        filename = os.path.join('thirdparty', 'PUI')
```

Comment:
We should make the consumption of PUI more systematic in the future, so that we do not need to hard-code import path like here.  While we can live with it now, @j8xixo12 could you please create an issue to track the improvement for this?